data-raw/create_covid_data.R

```diff
@@ -1,22 +1,17 @@
 # Script to generate large covid dataset with multiple age groups.
 # Run this script when the data needs to be updated
-if (!requireNamespace("readr", quietly = TRUE)) {
```

Comment:
Minor point, but should we rename this script `germany_covid19_hosp.R` for consistency with naming the downward correction script the name of the data?